orc8r/cloud/go/services/state/indexer/reindex/reindexer_singleton_test.go

```diff
@@ -123,27 +124,47 @@ func TestSingletonRun(t *testing.T) {
 	fail3.AssertExpectations(t)
 	require.Equal(t, 2, reindexSuccessNum)
 	require.Equal(t, 5, reindexDoneNum)
+
+	// Test that a network/hardware pair that has been added after Run
+	// will have its states reindexed as well
```

Comment:
Add a newline under this comment, for clarity that it applies to the block rather than the line